.clang-format

```diff
@@ -58,8 +58,8 @@ IndentCaseLabels: true
 IndentGotoLabels: false
 #llvm11: IndentExternBlock: AfterExternBlock
 #llvm11: InsertTrailingCommas: None
-MacroBlockBegin: "^BEGIN_FUNC"
```

Comment:
The BEGIN_FUNC/END_FUNC macros were removed a while ago.  So, I actually removed them and added the _BEFORE_USER_CB/_AFTER_USER_CB macros.